src/sentry/relay/config/metric_extraction.py

```diff
@@ -310,7 +307,6 @@ def _convert_snuba_query_to_metrics(
     environment = snuba_query.environment.name if snuba_query.environment is not None else None
     return _convert_aggregate_and_query_to_metrics(
         project,
-        snuba_query.dataset,
```

Comment:
Not need anymore.
<img width="545" alt="image" src="https://github.com/getsentry/sentry/assets/44410/514df61b-9c05-4aab-a0a9-b2a9923c8559">